fdbserver/workloads/ApiCorrectness.actor.cpp

```diff
@@ -26,15 +26,7 @@
 #include "flow/actorcompiler.h"  // This must be the last #include.
 
 //An enum of API operation types used in the random test
-enum OperationType {
-	SET,
-	GET,
-	GET_RANGE,
-	GET_RANGE_SELECTOR,
-	GET_KEY,
-	CLEAR,
-	CLEAR_RANGE
-};
+enum OperationType { SET, GET, GET_RANGE, GET_RANGE_SELECTOR, GET_KEY, CLEAR, CLEAR_RANGE, UNINITIALIZED };
```

Comment:
Not super happy about having introduced a new element to this enum, but it seems clear enough to me